File: logica-programacao/javascript/010.js
/* Dado um número inteiro positivo n com pelo menos dois dígitos:

Obtenha somente o penúltimo dígito

Exemplo:
n = 5729 → resultado = 2 */ 

let numeroInteiro = Number(prompt("Digite um número com pelo menos dois dígitos: "));

let semUltimo = Math.floor(numeroInteiro / 10);
let penultimoNumero = semUltimo % 10;

console.log(`O número ${numeroInteiro} tem como penúltimo ${penultimoNumero}.`);

File: logica-programacao/python/010.py
"""
Dado um número inteiro positivo n com pelo menos dois dígitos:

Obtenha somente o penúltimo dígito

Exemplo:
n = 5729 → resultado = 2

"""

numero_inteiro = int(input("Digite um número com pelo menos dois digitos: "))

sem_ultimo = numero_inteiro // 10
penultimo_numero = sem_ultimo % 10

print(f"O número {numero_inteiro} tem como penúltimo {penultimo_numero}.")